spec/models/post_mover_spec.rb

```diff
@@ -2463,5 +2463,24 @@ def create_topic_user(user, topic, opts = {})
         end
       end
     end
+
+    describe "with event trigger" do
+      fab!(:topic_1) { Fabricate(:topic) }
+      fab!(:topic_2) { Fabricate(:topic) }
+      fab!(:post_1) { Fabricate(:post, topic: topic_1) }
+      fab!(:post_2) { Fabricate(:post, topic: topic_1) }
+
+      it "uses first_post_moved trigger for first post" do
+        post_mover = PostMover.new(topic_1, Discourse.system_user, [post_1.id])
+        events = DiscourseEvent.track_events(:first_post_moved) { post_mover.to_topic(topic_2.id) }
```

Comment:
I think it is important to test that the right parameters are passed to the event as well.